marklogic-data-hub-central/ui/e2e/cypress/support/pages/browse.tsx

```diff
@@ -165,6 +170,19 @@ class BrowsePage {
    return cy.get('#selected-facets [data-cy="clear-'+ lowerBound +'"]');
   }
 
+  clickPopoverSearch(facetName: string){
+      return cy.get(`[data-testid=${facetName}-search-input]`).click();
+  }
+
+  getInputField(facetName: string,str: string){
+   return cy.get(`[data-testid=${facetName}-popover-input-field]`).type(str);
```

Comment:
We are setting the input field, so its better to have the method name reflect that. Lets call it setInputField.
can we use cy.findByTestId here and in the locator below.

Will the input field always be empty. Should we clear first before typing?